docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -0,0 +1,103 @@
+[role="xpack"]
+[[dfa-regression]]
+== {regression-cap}
+
+{reganalysis-cap} is a {ml} process for estimating the relationships among 
+different variables, then making further predictions based on these 
```

Comment:
Can we talk about relationships between fields in the data, instead of variables?
```suggestion
different fields in your data, then making further predictions based on these 
```